src/azure/cli/_help.py

```diff
@@ -386,5 +386,13 @@ def _get_single_metadata(cmd_table):
     assert len(cmd_table) == 1
     return next(metadata for _, metadata in cmd_table.items())
 
+def _get_group_priority(group_name):
+    priorities = {
+        'Resource Id Arguments': 10,
+        'Global Arguments': 1000,
+        }
```

Comment:
yep, this is where the change would be made.  The default for other groups is at the top, alphabetical